epoch/api/channels_api_usage.md

```diff
@@ -100,27 +100,33 @@ in the chain, and the others are metadata used for the connection itself.
   We will be using the tool [wscat](https://github.com/websockets/wscat)
   We assume the channel's WebSocket listener is set on port 3014 (default one)
 
-### Initiator WebSocket open
-Using the set of prenegotiated parameters the initiator connects
+### Responder WebSocket open
+Using the set of prenegotiated parameters the responder connects
 ```
-$ wscat --connect
-'localhost:3014/channel?initiator_id=ak_4Kr76woCtc3ehZ45K1sCrmgKX6gnh7qGhjSU1GZYqfLTTjtCgn&responder_id=ak_35Wxqf2cbzQF5hEV1j9AdXQFTMxqKCwM7iMKNGcqSv47MXAj68&lock_period=10&push_amount=3&initiator_amount=10&responder_amount=10&channel_reserve=2&ttl=1000&host=localhost&port=1234&role=initiator'
+$ wscat --connect 'localhost:3014/channel?initiator_id=ak_8wWs1j2vhgjexQmKfgEBrG8ysAucRJdb3jsag3PJKjEeXswb7&responder_id=ak_bmtGbfP3SdPoJNZCQGjjzbKRje15J9CEcWYaL1gZyv2qEyiMe&lock_period=10&push_amount=3&initiator_amount=10&responder_amount=10&channel_reserve=2&ttl=1000&port=1234&role=responder'
 
 connected (press CTRL+C to quit)
 ```
 
-Note the `role=initiator` as it is specific
+Note the `role=responder` as it is specific. Note also that the `host` is missing - it is not required for the responder.
+The `port` being specified is the one the `responder`'s node will start
+listening for the `initiator`'s connection. If the `responder`'s node is
+behind a firewall or some port forwarding is done - this should be done before
+the initiator starts connecting as it will fail.
 
-### Responder WebSocket open
-Using the set of prenegotiated parameters the responder connects
+At this point the `responder` is listening on address `0.0.0.0` for the `initiator`'s
```

Comment:
Regarding the `0.0.0.0`, please check [this comment](https://github.com/aeternity/epoch/pull/1667#issuecomment-431045975)